tests/all_tests.py

```diff
@@ -118,14 +118,14 @@ def test(suppress_unit_tests, suppress_javascript_unit_tests, unittest_glob):
     if not suppress_unit_tests:
         print("\nUnit test result summary\n------------------------\n")
         if bad_tests:
-            print(("\nFailed the following unit tests.\n%s"
-                   % '\n'.join(bad_tests)))
+            print("\nFailed the following unit tests.\n%s"
+                  % '\n'.join(bad_tests))
 
         if missing_application_tests:
-            print(("\nFailed the following unit tests, in part or whole due "
-                   "to missing external applications.\nDepending on the "
-                   "Emperor features you plan to use, this may not be "
-                   "critical.\n%s" % '\n'.join(missing_application_tests)))
+            print("\nFailed the following unit tests, in part or whole due "
```

Comment:
Changed the following messages :P ... fixed in both cases, thanks!

On (Jun-28-16| 7:07), Antonio Gonzalez wrote:
> >  
> >          if missing_application_tests:
> > -            print(("\nFailed the following unit tests, in part or whole due "
> > -                   "to missing external applications.\nDepending on the "
> > -                   "Emperor features you plan to use, this may not be "
> > -                   "critical.\n%s" % '\n'.join(missing_application_tests)))
> > +            print("\nFailed the following unit tests, in part or whole due "
> 
> Same here, I think it reads oddly. 
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/509/files/ace65cb3fba1716834d0171d871a58fc746a3b40#r68762782